assets/js/Components/UfixitModal.js

```diff
@@ -87,14 +105,18 @@ class UfixitModal extends React.Component {
     const pending = (this.props.activeIssue && (this.props.activeIssue.pending == '1'))
 
     let activeIndex = this.findActiveIndex();
-    const UfixitForm = returnIssueForm(activeIssue)
 
+    let code = ''
+    let UfixitForm
     let showExample = false
     if (!this.props.t(`rule.example.${activeIssue.scanRuleId}`).includes('rule.example')) {
       showExample = true
     }
 
-    let code = this.prepareCode(activeIssue)
+    if (!this.state.issueDetailsPending) {
+      UfixitForm = ufixitService.returnIssueForm(activeIssue)
```

Comment:
I cannot find `ufixitService` anywhere in the React files, only on this line. Returns an error:
```
Uncaught (in promise) ReferenceError: ufixitService is not defined
    at a.value (app.js:252:268294)
    at Pa (app.js:377:69887)
    at Na (app.js:377:69682)
    at gs (app.js:377:105481)
    at cl (app.js:377:96728)
    at sl (app.js:377:96653)
    at Zs (app.js:377:93683)
    at app.js:377:45325
    at t.unstable_runWithPriority (app.js:11:3844)
    at Fi (app.js:377:45034)
```